core/src/main/java/org/jruby/truffle/translator/BodyTranslator.java

```diff
@@ -2036,8 +2163,19 @@ public RubyNode visitPostExeNode(PostExeNode node) {
     public RubyNode visitRationalNode(RationalNode node) {
         final SourceSection sourceSection = translate(node.getPosition());
 
-        // TODO: implement Rational
-        return new FixnumLiteralNode.LongFixnumLiteralNode(context, sourceSection, node.getNumerator());
+        // Translate as Rubinius.privately { Rubinius.convert(a, b) }
```

Comment:
I believe this is how Rubinius does it, but I actually couldn't find the code that does it. They've got a couple of layers of parser and translator and I can't find where it's done. Privately because `Rational#convert` is private - it's a Rubinius implementation detail.